Python/Product/Cookiecutter/View/CookiecutterOptionsPage.xaml.cs

```diff
@@ -76,7 +77,28 @@ class TemplateContextItemTemplateSelector : DataTemplateSelector {
             var element = container as FrameworkElement;
             var p = item as ContextItemViewModel;
             if (element != null && p != null) {
-                var templateName = p.Items.Count == 0 ? "textTemplate" : "listTemplate";
+                string templateName;
+                switch (p.ValueType) {
+                    case Model.ContextItemValueType.String:
+                        templateName = "textTemplate";
```

Comment:
What if we skip the enum and just pass through the selector name + "Template" as the template name? I'm just trying to minimize the changes necessary to add new ones later.